packages/hw-app-eth/src/Eth.ts

```diff
@@ -262,12 +267,28 @@ export default class Eth {
       offset += chunkSize;
     }
 
-    rlpTx = ethers.utils.RLP.decode("0x" + rawTxHex);
```

Comment:
@machard : do you know why we needed to prefix 0x in the past? Also do you know why we had to call it twice. I wonder if we could use a higher level function that would have avoided having to do the logic of that pr on our side. Because ethers must have somewhere the logic to extract the rlp part of a tx?